cmd/neofs-adm/internal/modules/config/config.go

```diff
@@ -8,7 +8,7 @@ import (
 	"text/template"
 
 	"github.com/nspcc-dev/neo-go/cli/input"
-	"github.com/nspcc-dev/neofs-node/pkg/innerring"
+	"github.com/nspcc-dev/neofs-node/pkg/util/glagolitsa"
```

Comment:
A nice change on its own, btw. Whole `innerring` import is certainly not needed here.